src/SDKs/SqlManagement/Sql.Tests/DatabaseRestoreScenarioTests.cs

```diff
@@ -163,104 +163,138 @@ public void TestDatabaseRestore()
         }
 
         [Fact]
-        public void TestLongTermRetentionCrud()
+        public void TestLongTermRetentionV2Policies()
         {
+            string defaultPolicy = "PT0S";
+
             using (SqlManagementTestContext context = new SqlManagementTestContext(this))
             {
                 ResourceGroup resourceGroup = context.CreateResourceGroup();
-                RecoveryServicesClient recoveryClient = context.GetClient<RecoveryServicesClient>();
-                RecoveryServicesBackupClient backupClient = context.GetClient<RecoveryServicesBackupClient>();
+                Server server = context.CreateServer(resourceGroup);
                 SqlManagementClient sqlClient = context.GetClient<SqlManagementClient>();
+                Database database = sqlClient.Databases.CreateOrUpdate(resourceGroup.Name, server.Name, SqlManagementTestUtilities.GenerateName(), new Database { Location = server.Location });
+
+                // Get the policy and verify it is the default policy
+                //
+                BackupLongTermRetentionPolicy policy = sqlClient.BackupLongTermRetentionPolicies.Get(resourceGroup.Name, server.Name, database.Name);
+                Assert.Equal(defaultPolicy, policy.WeeklyRetention);
+                Assert.Equal(defaultPolicy, policy.MonthlyRetention);
+                Assert.Equal(defaultPolicy, policy.YearlyRetention);
+                Assert.Equal(0, policy.WeekOfYear);
+
+                // Set the retention policy to two weeks for the weekly retention policy
+                //
+                BackupLongTermRetentionPolicy parameters = new BackupLongTermRetentionPolicy(weeklyRetention: "P2W");
+                var policyResult = sqlClient.BackupLongTermRetentionPolicies.CreateOrUpdateWithHttpMessagesAsync(resourceGroup.Name, server.Name, database.Name, parameters).Result;
+                sqlClient.GetPutOrPatchOperationResultAsync(policyResult, new Dictionary<string, List<string>>(), CancellationToken.None).Wait();
```

Comment:
Nit: not really necessary to use `WithHttpMessagesAsync` in e2e test. You can simplify by using:

`var policyResult = sqlClient.BackupLongTermRetentionPolicies.CreateOrUpdate(...)`